<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analisis Semg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>20</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="grafico1" native="true">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>9</y>
      <width>471</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="grafico2" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>471</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonCargar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboMetodos">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>80</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboFiltros">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Metodos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label0">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Se√±al</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtros</string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dato2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dato1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dato3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="text2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonGraficar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Graficar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
